src/types/vault-types.ts

```diff
@@ -80,7 +81,12 @@ export type OracleDailyPriceNodesData = { [key: string]: OraclePriceDailiesNode[
 export type VaultManagerGovernancesNodesData = { [key: string]: VaultManagerGovernancesNode };
 
 export type VaultsDashboardData = {
-  [key: string]: VaultManagerMetricsNode & OraclePriceNode & VaultManagerGovernancesNode & OraclePriceDailiesArr;
+  [key: string]: VaultManagerMetricsNode &
+    OraclePriceNode &
+    VaultManagerGovernancesNode &
+    OraclePriceDailiesArr & { decimalPlaces: number };
 };
 
-export type OpenVaultsData = Array<VaultsNode & OraclePriceNode & VaultManagerGovernancesNode>;
\ No newline at end of file
+export type OpenVaultsData = Array<
+  VaultsNode & OraclePriceNode & VaultManagerGovernancesNode & { decimalPlaces: number }
+>;
```

Comment:
why dont we use `BoardAuxesNode` instead of `{ decimalPlaces: number }`

Reply:
`BoardAuxesNode` has `allegedName` property as well. That is not required.
But now that you've mentioned it, I think we should use `Pick<BoardAuxesNode, 'decimalPlaces'>;` for clarity?